doc/orbits.xml

```diff
@@ -171,18 +198,22 @@ Group(())
 <ManSection>
   <Attr Name="DigraphOrbits" Arg="digraph"/>
   <Returns>
-    A list of lists of integers.
+    An immutable list of immutable lists of integers.
```

Comment:
The sublists in an immutable list are necessarily immutable (something obviously isn't immutable if it contains mutable things). So the second immutable is redundant here.